custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -421,7 +421,9 @@ def validate(self, dataframe):
             io_type = "Output"
 
         if any_missing and value in [missing_output_disallowed, missing_input_disallowed]:
-            return [f"{io_type} contains missing values, the task does not support missing."]
+            return [
+                f"{io_type} contains missing values, but the schema says that they are not allowed."
```

Comment:
```suggestion
                f"{io_type} contains missing values, which the supplied schema does not allow."
```

Reply:
How about
f"{io_type} contains missing values, which the supplied task schema does not allow."